src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -2356,4 +2356,33 @@ describe('Formulas general', () => {
       });
     });
   });
+
+  it('should not try to validate cells outside of the table boundaries', () => {
```

Comment:
Can I ask you to move the validation test type to the `/src/plugins/formulas/__tests__/validation.spec.js` spec file?

Reply:
Sure thing! fb9730b